ml-algorithms/src/main/java/org/opensearch/ml/engine/ModelHelper.java

```diff
@@ -138,6 +140,26 @@ public void downloadPrebuiltModelConfig(
                                     }
                                 }
                                 builder.modelConfig(configBuilder.build());
+                            } else if (FunctionName.IMAGE_EMBEDDING.equals(algorithm)) {
+                                ImageEmbeddingModelConfig.ImageEmbeddingModelConfigBuilder configBuilder = ImageEmbeddingModelConfig
+                                    .builder();
+                                Map<?, ?> configMap = (Map<?, ?>) entry.getValue();
+                                for (Map.Entry<?, ?> configEntry : configMap.entrySet()) {
+                                    switch (configEntry.getKey().toString()) {
+                                        case MLModelConfig.MODEL_TYPE_FIELD:
+                                            configBuilder.modelType(configEntry.getValue().toString());
+                                            break;
+                                        case MLModelConfig.ALL_CONFIG_FIELD:
+                                            configBuilder.allConfig(configEntry.getValue().toString());
+                                            break;
+                                        case ImageEmbeddingModelConfig.EMBEDDING_DIMENSION_FIELD:
+                                            configBuilder.embeddingDimension((Integer) configEntry.getValue());
```

Comment:
Handle exception when user not setting the value to number.

Reply:
This is handled in `ImageEmbeddingModelConfig`